src/wmnds/components/form-elements/input/_input.scss

```diff
@@ -8,4 +8,8 @@
   &--error {
     border: 2px solid get-color(error);
   }
+
+  &.wmnds-col-1-2 {
+    width: 50%;
+  }
 }
```

Comment:
This isn't needed, just wrap the html element with a <div class="wmnds-col-1-2"><input /></div>.